taxonomies/food/categories.txt

```diff
@@ -103813,7 +103813,13 @@ ciqual_proxy_food_name:en: Jam, strawberry
 ciqual_proxy_food_name:fr: Confiture de fraise (extra ou classique)
 
 < en:Berry jams
-en: Bilberries jams
+en: Blueberry jams
```

Comment:
Another approach could maybe be to call this something like “Generic blueberry jam” or something, and then the `en: American blueberry jams` could just be `en: Blueberry jams`?

```suggestion
en: Blueberry jams (generic)
```